setup.py

```diff
@@ -11,14 +16,18 @@
         'backports.shutil_get_terminal_size',
     )
 
+with open(join(ROOT, "README.md")) as fd:
+    README = fd.read()
 
 setup(
     name='adventurelib',
     description='Easy text adventures',
+    long_description=README,
+    long_description_content_type="text/markdown",
     version=__version__,
     author='Daniel Pope',
     author_email='mauve@mauveweb.co.uk',
-    url='https://adventurelib.readthedocs.io/',
+    url='https://github.com/lordmauve/adventurelib',
```

Comment:
There really needs to be an easy to find documentation link because this is what beginners will be looking for first. Perhaps move the documentation link to a `project_urls` field?

Reply:
Fixed, thanks for pointing out 🙂